.gitlab-ci-agent.yml

```diff
@@ -235,28 +235,6 @@ unit_tests:
   retry:
     max: 2
 
-unit_tests_cluster_agent:
-  stage: test
-  interruptible: true
-  needs:
-    - deps_deb
-  script:
-    - conda activate $CONDA_ENV
-    # We run inv deps again because /go/bin is not cached and we need binaries like golint, misspell
-    - inv deps
-    - inv -e agent.build --race --precompile-only --major-version $MAJOR_VERSION --python-runtimes $PYTHON_RUNTIMES
-    # TODO: check why formatting rules differ from previous step
-    - gofmt -l -w -s ./pkg ./cmd
-    # TODO: --python-runtimes "$PYTHON_RUNTIMES"
-    - inv -e rtloader.test
-    - inv -e test --coverage --race --profile --fail-on-fmt --cpus 4 --major-version $MAJOR_VERSION --python-runtimes $PYTHON_RUNTIMES --build-include docker,linux,kubelet,kubeapiserver,python,orchestrator,clusterchecks,zlib
-  tags:
-    - sts-aws
-  rules:
-    - *cluster_agent
-  retry:
-    max: 2
-
```

Comment:
Why is this deleted?  Are the cluster agent unit tests no longer relevant? Or have they perhaps been moved?

Reply:
we figured it was redundant, runs the same test as the unit_test step